tests/integration/install/test_installation.py

```diff
@@ -392,14 +393,29 @@ def test_check_inventory_database_exists(ws, installation_ctx):
         installation_ctx.workspace_installer.configure()
 
 
-def test_compare_remote_local_install_versions(installation_ctx) -> None:
-    installation_ctx.workspace_installation.run()
+def is_on_release_branch(checkout_root: str) -> bool:
+    """Check if running on a release branch"""
+    release_branches = {"prepare/"}
+    try:
+        out = subprocess.run(
+            ["git", "rev-parse", "--abbrev-ref", "HEAD"], stdout=subprocess.PIPE, check=True, cwd=checkout_root
+        )
+        current_branch = out.stdout.decode("utf8")
```

Comment:
I think you can also specify the encoding as a parameter for `subprocess.run()`.